### PR TITLE
Add documentation and examples for cross-account mount

### PR DESCRIPTION
**Pull request by kbasv**
**Is this a bug fix or adding new feature?**

**What is this PR about? / Why do we need it?**
Adds documentation for #470 

**What testing is done?**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -34,10 +34,12 @@ The following CSI interfaces are implemented:
 | gidRangeStart       |        | 50000   | true      | Start range of the POSIX group Id to be applied for [Access Point root directory](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html#enforce-root-directory-access-point) creation. |
 | gidRangeEnd         |        | 7000000 | true      | End range of the POSIX group Id. |
 | basePath            |        |         | true      | Path under which access points for dynamic provisioning is created. If this parameter is not specified, access points are created under the root directory of the file system |
+| az                  |        |         | true      | Used for cross-account mount. `az` under storage class parameter is optional. If specified, mount target associated with the az will be used for cross-account mount. If not specified, a random mount target will be picked for cross account mount |
```
Comment (wongma7) [inline docs/README.md:37]: default should be "" not false

File: examples/kubernetes/cross_account_mount/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  provisioningMode: efs-ap
+  fileSystemId: fs-1234abcd
+  directoryPerms: "700"
+  gidRangeStart: "1000"
+  gidRangeEnd: "2000"
+  basePath: "/dynamic_provisioning"
+  az: us-east-1
+```
+* provisioningMode - The type of volume to be provisioned by efs. Currently, only access point based provisioning is supported `efs-ap`.
+* fileSystemId - The file system under which Access Point is created.
```
Comment (wongma7) [inline examples/kubernetes/cross_account_mount/README.md:26]: I don't think it's necessary to copy the parameter docs from READMe, they may become out of sync

File: examples/kubernetes/cross_account_mount/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* directoryPerms - Directory Permissions of the root directory created by Access Point.
+* gidRangeStart (Optional) - Starting range of Posix Group ID to be applied onto the root directory of the access point. Default value is 50000. 
+* gidRangeEnd (Optional) - Ending range of Posix Group ID. Default value is 7000000.
+* basePath (Optional) - Path on the file system under which access point root directory is created. If path is not provided, access points root directory are created under the root of the file system.
+* `az` under storage class parameter is optional. If specified, mount target associated with the az will be used for cross-account mount. If not specified, a random mount target will be picked for cross account mount
+
+### Prerequisite setup
+1.  Perform [vpc-peering](https://docs.aws.amazon.com/vpc/latest/peering/working-with-vpc-peering.html) between EKS cluster `vpc` and EFS `vpc`.
```
Comment (wongma7) [inline examples/kubernetes/cross_account_mount/README.md:34]: the cluster is in account A and EFs in B? or other way around

File: examples/kubernetes/cross_account_mount/README.md (hunk 2 above)
Comment (kbasv) [reply examples/kubernetes/cross_account_mount/README.md:26]: Yeah, will get rid of the parameters

File: examples/kubernetes/cross_account_mount/README.md (hunk 3 above)
Comment (kbasv) [reply examples/kubernetes/cross_account_mount/README.md:34]: Cluster is in account A and efs in account B. Let me update it here for clarity.

File: docs/README.md (hunk 1 above)
Comment (kbasv) [reply docs/README.md:37]: Updated.

**Comment by wongma7** [comment]
/lgtm
/hold

want the code to merge first

**Pull request merged**